### PR TITLE
Made the comment line more comprehensive

### PR DESCRIPTION
**Pull request by anusha-exe**

### CODE REVIEW

**Comment by anusha-exe** [comment]
Awesome work but should have added some comment lines describing the code!

File: sixteenSegmentDisplay.pde [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// This programme based on anusha-exe's seven segment display LED clock programme
+// This programme is based on The Coding Train video on Seven Segment Display
```
Comment (DenisovichDev) [inline sixteenSegmentDisplay.pde:1]: I really don't agree with this particular line, good lady. So if you fix it, I can pull it.

**Comment by DenisovichDev** [comment]
Yeah, but I'll probably change the acknowledgements sections
.

**Comment by anusha-exe** [comment]
You know. I have no idea what you're doing, but okay

**Pull request merged**